lib/ui/order/order_page.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:mskirana_app/ui/confirmorder/confirm_order_page.dart';
+import 'package:mskirana_app/ui/common/shared_bottom_bar.dart';
+import 'package:mskirana_app/ui/order/order_item_view.dart';
+
+final keyItem = new GlobalKey<OrderItemViewState>();
```

Comment:
Same, where is this being used?

Reply:
this is used to get the current state in case of all pages, while passing it to confirm order.

Reply:
Oh, interesting pattern! Nice.